src/utils/StatsigFetcher.ts

```diff
@@ -115,6 +116,7 @@ export default class StatsigFetcher {
         'STATSIG-SDK-TYPE': getSDKType(),
         'STATSIG-SDK-VERSION': getSDKVersion(),
       },
+      signal: options?.signal,
```

Comment:
This is the important part, because this line will go all the way into [safeFetch](https://github.com/statsig-io/node-js-server-sdk/blob/8b56c8c1d8552aa6beb30c83c7783edf2673a0a4/src/utils/safeFetch.ts#L13), which can handle the signal because it's just a normal `fetch()` under the hood.